src/software/logger/coloured_cout_sink.h

```diff
@@ -35,6 +35,9 @@ class ColouredCoutSink
      */
     void displayColouredLog(g3::LogMessageMover log_entry);
 
+    std::chrono::_V2::system_clock::duration LOG_INTERVAL_TIMESTAMP =
```

Comment:
can we rename this to something more descriptive something like LOG_MERGE_DURATION